openmdao/docs/features/building_components/explicitcomp.rst

```diff
@@ -16,6 +16,13 @@ ExplicitComponent Methods
 
 The implementation of each method will be illustrated using a simple explicit component that computes the output *area* as a function of inputs *length* and *width*.
 
+::
```

Comment:
I was actually just thinking of doing this myself. By the way, isn't there an automatic way of doing this? Along the lines of:
```
.. autoclass:: openmdao.core.group.Group
    :noindex:
```